src/factoids.coffee

```diff
@@ -43,22 +43,22 @@ module.exports = (robot) ->
       to ?= msg.message.user.name
       msg.send "#{to.trim()}: #{fact.value}"
 
-  robot.respond new RegExp("[#{prefix}]([\\w\\s-]{2,}\\w)", 'i'), (msg) =>
+  robot.respond new RegExp("[#{prefix}]([\\w\\s-]{1,}\\w)", 'i'), (msg) =>
     fact = @factoids.get msg.match[1]
     if not fact? or fact.forgotten
       msg.reply "Not a factoid"
     else
       fact.popularity++
       msg.send "#{fact.value}"
 
-  robot.respond /learn (.{3,}) = ([^@].+)/i, (msg) =>
+  robot.respond /learn (.{2,}) = ([^@].+)/i, (msg) =>
```

Comment:
These should probably be updated to `(\w{2,})` yeah? I can do it in a different PR if you don't want to bother, but I figure since you're updating and I'm thinking about it, couldn't hurt. Up to you.